src/policy/policy_deny_fs.cc

```diff
@@ -0,0 +1,137 @@
+#include "policy_deny_fs.h"
+#include "base_object-inl.h"
+#include "v8.h"
+
+#include <fcntl.h>
+#include <limits.h>
+#include <stdlib.h>
+#include <filesystem>
+#include <iostream>
+#include <string>
+#include <vector>
+
+using v8::Just;
+using v8::Maybe;
+
+namespace node {
+
+namespace policy {
+
+// deny = 'fs'
+// deny = 'in:/tmp/'
+// deny = 'in:/tmp/,out:./example.js'
+Maybe<bool> PolicyDenyFs::Apply(const std::string& deny) {
+  for (const auto& name : SplitString(deny, ',')) {
+    Permission perm = Permission::kPermissionsRoot;
+    for (std::string& opt : SplitString(name, ':')) {
```

Comment:
What if a path contains literal `,` or `:` characters?

Reply:
IMO That wouldn't be supported in that first iteration. We haven't fully decided about that nomenclature, that's the reason I didn't fill up edge cases.